config/api.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: 'https://reqres.in/api'
```

Comment:
In a real app we would have many environments (development, staging, production) so the base api url should change for each environment. That's why baseURL should be and environment variable.